docs/docs/upgrading-node-js.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Upgrading your Node.js version
+---
+
+## Gatsby's Node.js support policy
+
+Gatsby aims to support any version of Node that has a release status of _Current_, _Active_, or _Maintenance_. Once a major version of Node reaches _End of Life_ status Gatsby will stop supporting that version.
+
+Gatsby will stop supporting the _End of Life_ Node release in a minor version.
+
+Check [Node's releases document](https://github.com/nodejs/Release#nodejs-release-working-group) for version statuses.
+
+## What version of Node do I have?
+
+Run `node -v` in a terminal to see which version of Node you have.
+
+```
+node -v
+v10.16.0
+```
+
+This example shows Node version 10, specifically v10.16.0.
+
+## Upgrading from Node version 6
+
+Node version 6 reached _End-of-life_ status on 30th April 2019. Many of Gatsby's dependencies are updating to Node version 8 and above. Gatsby must also update in order to deliver new features and bug fixes more quickly.
+
+Generally we'd recommend using [the Node version whose status is _Active LTS_](https://github.com/nodejs/Release#nodejs-release-working-group) (Node 10 at time of writing). However, in this document you'll learn how to update from Node 6 to Node 8 as this is likely to be the least disruptive upgrade for you.
+
+> What about Node 7? Stable versions of Node are evenly numbered releases - Node 6, Node 8, Node 10 etc. Only use uneven release numbers if you'd like to try cutting-edge and experimental features.
+
+There are multiple ways to update your version of Node depending on how you originally installed it. Read on to find the best approach for you.
+
+### Using Homebrew
+
+This is our recommended way to install a newer version of Node.
```

Comment:
This applies to OSX only, but it's not mentioned anywhere

Reply:
It's on Linux and Windows (subsystem for Linux) too! See https://www.gatsbyjs.org/tutorial/part-zero/#install-homebrew-for-nodejs

Reply:
Oh, TIL